packages/sdk/test/common/setup.ts

```diff
@@ -0,0 +1,10 @@
+import { afterAll } from "vitest";
+
+// optionally reset anvil instance to the initial fork block.
+//  commenting this out since we don't do chain interactions in every test file
+afterAll(async () => {
+  //   await Promise.all(
+  //     Object.values(chainClients).map((client) => client.reset()),
+  //   );
```

Comment:
ooc, does this take a lot of time to run? Just wondering if it's worth having it enabled by default even though we technically don't need it.

Reply:
It does take long, but actually these all run in parallel, so each non-anvil test file will still run shorter than the longest one. Will uncomment this, I agree its better to be safe 👍